ertviz/controllers/plot_view_controller.py

```diff
@@ -0,0 +1,87 @@
+from dash.dependencies import Input, Output, State
+from dash.exceptions import PreventUpdate
+import dash_bootstrap_components as dbc
+import dash_html_components as html
+from ertviz.views import response_view, parameter_view
+from .multi_parameter_controller import multi_parameter_controller
+from .multi_response_controller import multi_response_controller
+
+
+def _get_child(response, children):
+    for child in children:
+        if child["props"]["children"][0]["props"]["data"] == response:
+            return child
+    return None
+
+
+def plot_view_controller(parent, app):
+    multi_parameter_controller(parent, app)
+    multi_response_controller(parent, app)
+
+    @app.callback(
+        Output(parent.uuid("plot-selection-store"), "data"),
+        [
+            Input(parent.uuid("parameter-selector"), "value"),
+            Input(parent.uuid("response-selector"), "value"),
+        ],
+        [State(parent.uuid("plot-selection-store"), "data")],
+    )
+    def update_plot_selection(parameters, responses, current_selection):
+        parameters = [] if not parameters else parameters
+        responses = [] if not responses else responses
+        current_selection = [] if not current_selection else current_selection
+        for plot in current_selection.copy():
```

Comment:
How does this work? Is the copy created upfront or every cycle a new copy?

Reply:
Only in the beginning